samples/star/src/jvmMain/kotlin/com/slack/circuit/star/common/BackPressNavIcon.jvm.kt

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) 2022 Slack Technologies, LLC
+// SPDX-License-Identifier: Apache-2.0
+package com.slack.circuit.star.common
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+
+@Composable
+actual fun BackPressNavIcon(
+  modifier: Modifier,
+  onClick: (() -> Unit)?,
+  iconButtonContent: @Composable () -> Unit,
+) {
+  // We do nothing on desktop
+  // TODO maybe we do escaping?
```

Comment:
We'd need to do something that sends an escape event to the window, as that's sorta how I've implemented backstack popping here

Reply:
Think we'll have to create a back press local so that it can get triggered correctly on the window 🤔

Reply:
yeah, I think it's out of scope for this PR as a start anyway